packages/lesswrong/components/posts/EAPostsItem.tsx

```diff
@@ -0,0 +1,297 @@
+import React, { FC, MouseEvent } from "react";
+import { registerComponent, Components } from "../../lib/vulcan-lib";
+import { AnalyticsContext } from "../../lib/analyticsEvents";
+import { usePostsItem, PostsItemConfig } from "./usePostsItem";
+import { SoftUpArrowIcon } from "../icons/softUpArrowIcon";
+import { HashLink } from "../common/HashLink";
+import { Link, useHistory } from "../../lib/reactRouterWrapper";
+import { SECTION_WIDTH } from "../common/SingleColumnSection";
+import withErrorBoundary from "../common/withErrorBoundary";
+import classNames from "classnames";
+
+export const styles = (theme: ThemeType): JssStyles => ({
+  root: {
+    position: "relative",
+    maxWidth: SECTION_WIDTH,
+    display: "flex",
+    alignItems: "center",
+    background: theme.palette.grey[0],
+    border: `1px solid ${theme.palette.grey[100]}`,
+    borderRadius: theme.borderRadius.default,
+    padding: `8px 12px 8px 0`,
+    fontFamily: theme.palette.fonts.sansSerifStack,
+    fontWeight: 500,
+    fontSize: 14,
+    color: theme.palette.grey[600],
+    cursor: "pointer",
+    [theme.breakpoints.down("xs")]: {
+      paddingRight: 12,
+    },
+    "&:hover": {
+      background: theme.palette.grey[50],
+      border: `1px solid ${theme.palette.grey[250]}`,
+    },
+    "&:hover .PostsItemTrailingButtons-actions": {
+      opacity: 0.2,
+    },
+    "&:hover .PostsItemTrailingButtons-archiveButton": {
+      opacity: 0.2,
+    },
+  },
+  karma: {
+    width: 50,
+    minWidth: 50,
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+  },
+  tagRelWrapper: {
+    position: "relative",
+    marginLeft: 30,
+  },
+  voteArrow: {
+    color: theme.palette.grey[400],
+    margin: "-6px 0 2px 0",
+  },
+  details: {
+    flexGrow: 1,
+    minWidth: 0, // flexbox black magic
+  },
+  title: {
+    fontWeight: 650,
```

Comment:
```suggestion
    fontWeight: 600,
```
Changing back to 600 after a few people mentioned they feel too bold